cpp/modules/deck.gl/layers/test/line-layer-test.cpp

```diff
@@ -108,7 +108,7 @@ TEST_F(LineLayerTest, Create) {
 }
 
 TEST_F(LineLayerTest, GetSourcePositionData) {
-  // TODO(ilija@unfolded.ai): Props should be mocked in order for this test to be decoupled and make sense
+  // TODO(ilija@unfolded.ai): Props should be mocked in order to completely decouple this test from props implementation
```

Comment:
Nit: To be frank, I don't think that level of decoupling makes much sense. The props class is pretty trivial and unit testing a layer and its props together is fine. I'd drop these comments.